app/views/pages/index.html.erb

```diff
@@ -59,8 +61,9 @@
         <%= link_to "View reading list", '/articles', class: "glow-primary no-margin btn btn-large btn-success internal-link" %>
         <div class="faded text-small">(you're already logged in and ready to go) 🚀</div>
       <% else %>
-        <%= link_to "Create an account", '/signup', method: :post, class: "glow-primary no-margin btn btn-large btn-success internal-link" %>
-        <div class="faded text-small">(literally takes a second, no email required—really, try it!)</div>
+        <%= link_to t(:create_an_account), '/signup', method: :post, class: "glow-primary no-margin btn btn-large btn-success internal-link" %>
+        <div class="faded text-small"></div>
+        <%= t(:no_email_required) %>
```

Comment:
Looks like this string is now outside the `<div>` tag, we should keep it inside:

```suggestion
        <div class="faded text-small"><%= t(:no_email_required) %></div>
```